native/acter/src/api/news.rs

```diff
@@ -634,8 +637,14 @@ impl NewsEntryDraft {
         trace!("starting send");
         let client = self.client.clone();
         let room = self.room.clone();
+        let my_id = room
+            .client()
```

Comment:
we already have client
```suggestion
        let my_id = client
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1671/commits/1a56d9a426d303deef8dfdcdb08c5c870f3a6716